### PR TITLE
Chart

### PR DESCRIPTION
**Pull request by geoffrey1330**

### CODE REVIEW

File: src/components/ChartsViz.js [hunk 1]
```diff
@@ -1,17 +1,26 @@
 import React from 'react'
 import Chart from './Chart'
 
-export default function ChartsViz({chartComp}) {
+export default function ChartsViz({chartComp,setChartComp}) {
+  const remover = (key) => {
+    setChartComp(chartComp.filter((el) => el.key !==key));
```
Comment (steveoni) [inline src/components/ChartsViz.js:6]: instead of `filter` use `splice()` to delete the chart component.

File: src/components/DataTables.js [hunk 2]
```diff
@@ -1,19 +1,28 @@
 import React from 'react'
 import DataTable from './DataTable'
 
-export default function DataTables({datacomp, setCompIndex, setSidePlane}) {
-
+export default function DataTables({datacomp, setCompIndex, setSidePlane,setDataComp}) {
+  const remover = (keys) => {
+    setDataComp(datacomp.filter((el) => el.keys !==keys));
```
Comment (steveoni) [inline src/components/DataTables.js:6]: same thing here instead of filter we can delete the element totally.

File: src/App.js [hunk 3]
```diff
@@ -17,8 +17,8 @@ function App() {
   const changeHandler = function(event) {
       const content = event.target.files[0]
       const url = URL.createObjectURL(content)
-
-      read_csv(url, {start: 0, end: 20}).then(df => {
+      
+      read_csv("https://raw.githubusercontent.com/plotly/datasets/master/finance-charts-apple.csv", {start: 0, end: 20}).then(df => {
```
Comment (steveoni) [inline src/App.js]: remove this link and add back the default `url`  as `read_csv(URL)` before pushing

**Review by steveoni — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -7,3 +7,5 @@ Clone the repo and cd into the repo directory and then run `yarn`
 This a work in progres and lack some styling. The major part of `Data-aRT` is fully implemented here is what it looks like (in simple wireframe design):
 
 ![](exxample.png)
+
+![](Capture.PNG)
```
Comment (steveoni) [inline README.md]: remove `` from here add it to your pull request message

Comment (geoffrey1330) [reply README.md]: done

File: src/App.js (hunk 3 above)
Comment (geoffrey1330) [reply src/App.js]: should it be capital URL?

File: src/components/DataTables.js (hunk 2 above)
Comment (geoffrey1330) [reply src/components/DataTables.js:6]: i did filter so the component can be deleted individually

Comment (steveoni) [reply src/components/DataTables.js:6]: I understand but this not removing the element from the component state. can you try creating multiple charts and see if the deleting will work

File: src/App.js (hunk 3 above)
Comment (steveoni) [reply src/App.js]: no the former variable before adding this

Comment (geoffrey1330) [reply src/App.js]: ok

Comment (geoffrey1330) [reply src/App.js]: have you resolve the error in the danfo js lib

File: src/components/DataTables.js (hunk 2 above)
Comment (geoffrey1330) [reply src/components/DataTables.js:6]: it does

**Review by steveoni — APPROVED** [review]

**Pull request merged**